comid-code-points.cddl

```diff
@@ -19,101 +18,52 @@ comid.role = 2
 comid.linked-tag-id = 0
 comid.tag-rel = 1
 
-; element-name-map
-comid.label = 0
+; triples-map
+comid.reference-triples = 0
+comid.endorsed-triples = 1
+comid.identity-triples = 2
+comid.attest-key-triples = 3
+
+; environment-map
+comid.class = 0
+comid.instance = 1
+comid.group = 2
+
+; class-map
+comid.class-id = 0
 comid.vendor = 1
-comid.class-id = 2
-comid.model = 3
-comid.layer = 4
-comid.index = 5
+comid.model = 2
+comid.layer = 3
+comid.index = 4
 
-; element-value-map
-comid.version = 0
+; measurement-map
+comid.ver = 0
```

Comment:
Perhaps a comment on clarifying what does the comid.ver implies as against comid.version below, might help to a reader!

Reply:
I expect the text in the draft to explain such things.  Besides the comments are removed when the final CDDL is coalesced.